src/models/dft.js

```diff
@@ -1,107 +1,78 @@
-import _ from "underscore";
-import { treeSlugToNamedObject } from "../tree";
-import { Model } from "../model";
 import { MethodFactory } from "../methods/factory";
+import { UnitModel } from "../unit_model";
+import { Functional } from "./dft/functional";
+import { FUNCTIONALS } from "./dft/functional_lookup_table";
 
-export class DFTModel extends Model {
-
+export class DFTModel extends UnitModel {
     constructor(config) {
         super(config);
         this._MethodFactory = config.MethodFactory || MethodFactory;
+        this._functional = Functional.create({ slug: config.functional });
     }
 
     /**
      * @summary Build slug string based on model information
      * @returns {string}
      */
     get groupSlug() {
-        return [
-            this._application.shortName,
-            this.type,
-            this.subtype,
-            this.functional.slug,
-            this.refiners.map(o => o.slug).join("+"),
-            this.modifiers.map(o => o.slug).join("+"),
-        ].join(":").replace("::", ":").replace(/:$/, "");
-    }
-
-    get defaultFunctional() {
-        return treeSlugToNamedObject(this.treeBranchForSubType.functionals[0])
+        return [this._application.shortName, this.type, this.subtype, this.functional]
```

Comment:
When changing this, we will need to write a migration

Reply:
Ok I will add a todo item for the migration to the epic